fixtures/search-syntax/numeric_filter.json

```diff
@@ -3,92 +3,167 @@
     "desc": "Numeric format should still return a string if field isn't allowed",
     "query": "random_field:>500",
     "result": [
-      {"type": "spaces", "value": ""},
+      {
+        "type": "spaces",
+        "value": ""
+      },
       {
         "type": "filter",
         "filter": "text",
         "negated": false,
-        "key": {"type": "keySimple", "value": "random_field", "quoted": false},
+        "key": {
+          "type": "keySimple",
+          "value": "random_field",
+          "quoted": false
+        },
         "operator": "",
-        "value": {"type": "valueText", "value": ">500", "quoted": false}
+        "value": {
+          "type": "valueText",
+          "value": ">500",
+          "quoted": false
+        }
       },
-      {"type": "spaces", "value": ""}
+      {
+        "type": "spaces",
+        "value": ""
+      }
     ]
   },
   {
     "desc": "Numeric format should not negate operator if field isn't allowed",
     "query": "!random_field:>500",
     "result": [
-      {"type": "spaces", "value": ""},
+      {
+        "type": "spaces",
+        "value": ""
+      },
       {
         "type": "filter",
         "filter": "text",
         "negated": true,
-        "key": {"type": "keySimple", "value": "random_field", "quoted": false},
+        "key": {
+          "type": "keySimple",
+          "value": "random_field",
+          "quoted": false
+        },
         "operator": "",
-        "value": {"type": "valueText", "value": ">500", "quoted": false}
+        "value": {
+          "type": "valueText",
+          "value": ">500",
+          "quoted": false
+        }
       },
-      {"type": "spaces", "value": ""}
+      {
+        "type": "spaces",
+        "value": ""
```

Comment:
Do you need to rebase on top of the prettier update?

Reply:
turns out these were due to my viscose defaulting to another json formatting extension instead of prettier, will revert them

Reply:
👍